superset-frontend/src/featureFlags.ts

```diff
@@ -43,8 +43,40 @@ export enum FeatureFlag {
   ENABLE_EXPLORE_DRAG_AND_DROP = 'ENABLE_EXPLORE_DRAG_AND_DROP',
 }
 
+export type ScheduleQueriesProps = {
+  JSONSCHEMA: {
+    [key: string]: string;
+  };
+  UISCHEMA: {
+    [key: string]: string;
+  };
+  VALIDATION: {
+    [key: string]: string;
+  };
+};
 export type FeatureFlagMap = {
-  [key in FeatureFlag]?: boolean;
+  SCHEDULED_QUERIES?: ScheduleQueriesProps;
+  ALLOW_DASHBOARD_DOMAIN_SHARDING?: boolean;
+  OMNIBAR?: boolean;
+  CLIENT_CACHE?: boolean;
+  DYNAMIC_PLUGINS?: boolean;
+  SQL_VALIDATORS_BY_ENGINE?: boolean;
+  ESTIMATE_QUERY_COST?: boolean;
+  SHARE_QUERIES_VIA_KV_STORE?: boolean;
+  SQLLAB_BACKEND_PERSISTENCE?: boolean;
+  THUMBNAILS?: boolean;
+  LISTVIEWS_DEFAULT_CARD_VIEW?: boolean;
+  ENABLE_REACT_CRUD_VIEWS?: boolean;
+  DISABLE_DATASET_SOURCE_EDIT?: boolean;
+  DISPLAY_MARKDOWN_HTML?: boolean;
+  ESCAPE_MARKDOWN_HTML?: boolean;
+  DASHBOARD_NATIVE_FILTERS?: boolean;
+  DASHBOARD_CROSS_FILTERS?: boolean;
+  DASHBOARD_NATIVE_FILTERS_SET?: boolean;
+  VERSIONED_EXPORT?: boolean;
+  GLOBAL_ASYNC_QUERIES?: boolean;
+  ENABLE_TEMPLATE_PROCESSING?: boolean;
+  ENABLE_EXPLORE_DRAG_AND_DROP?: boolean;
```

Comment:
You should be able to do this instead: 
```
export type FeatureFlagMap = {
  [key in Exclude<FeatureFlag, FeatureFlag.SCHEDULED_QUERIES>]: boolean;
} & {
  SCHEDULED_QUERIES: ScheduleQueriesProps;
};
```

Reply:
this looks way better, thank you so much!